### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by adriaanbd**
This PR has the following:
* create react app boilerplate
* index.js and index.html
* App.js and index.js
* redux and react-redux dependencies
* eslint dependencies with airbnb config and default eslintrc.json file from Microverse [here](https://github.com/microverseinc/linters-config/tree/master/react-redux)
* stickler.yml

Remember to:

```
$ npm install
$ npm run start
```

My Stickler-CI is enabled:
image

Notes:
* The stickler file is present, and the repo is authorized. I don't know why the stickler bot isn't working.
* I'm authorized to work alone in this project.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "plugins": [
+      "react"
+  ],
+  "rules": {
+      "react/jsx-filename-extension": [
+          "warn", { "extensions": [".js", ".jsx"] }
+      ]
+  }
+}
```
Comment (bolah2009) [inline .eslintrc.json]: Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

image

See https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: .stickler.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+linters:
+  eslint:
+    config: './.eslintrc.json'
```
Comment (bolah2009) [inline .stickler.yml:4]: Let's make sure `stickler` is ON for this repo.

- Goto https://stickler-ci.com/
- Login
- ON the linter for the repo

image

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @adriaanbd 

Please see comments below on suggestions on how to fix the stickler. Please submit another PR even if the issue persists.

#### :recycle:  Make these changes and submit another code review request. 👍🏾

#### Feel free to reach me on slack if you having issues implementing these features. :relaxed: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: .stickler.yml (hunk 2 above)
Comment (adriaanbd) [reply .stickler.yml:4]: Bola, if you look at the screenshot I shared at the end of my PR comment, you'll notice that Stickler-CI was (and still is) enabled for this repository. Here is another screenshot taken 5 minutes ago:

image

Stickler simply doesn't work on this repo, and I don't know why.

File: .eslintrc.json (hunk 1 above)
Comment (adriaanbd) [reply .eslintrc.json]: I did not push my last changes and this file lacks:

```
      "import/no-unresolved": "off",
      "no-shadow": "off"
```

Thanks

**Comment by adriaanbd** [comment]
Please check `.eslintrc.json`'s last commit [here](https://github.com/adriaanbd/magicbooks/pull/1/commits/123bef7962f097a7c3e1b31d8bb6778237bca9af) which is the last commit of this branch. Bola kindly pointed out that it was incomplete, and he was right. The issue was addressed in the before-mentioned commit.

Here is Microverse's required linter config:

image

And here is mine:

```

{
  "env": {
      "browser": true,
      "es6": true,
      "jest": true
  },
  "extends": [
      "plugin:react/recommended",
      "airbnb"
  ],
  "parserOptions": {
      "ecmaFeatures": {
          "jsx": true
      },
      "ecmaVersion": 2018,
      "sourceType": "module"
  },
  "plugins": [
      "react"
  ],
  "rules": {
      "react/jsx-filename-extension": [
          "warn", { "extensions": [".js", ".jsx"] }
      ],
      "import/no-unresolved": "off",
      "no-shadow": "off"
  }
}
```

Furthermore, running ESLint locally shells no errors.

File: .stickler.yml (hunk 2 above)
Comment (bolah2009) [reply .stickler.yml:4]: Yes, Adriaan. Please see the difference between my screenshot and yours. Mine is ON and yours is OFF (hence the grey colour).

For stickler to be enabled. We need to enable it both on Github and on https://stickler-ci.com/

Also, if you ON it there would be no difference in the PR unless you push a commit to trigger it.

**Review by bolah2009 — APPROVED** [review]
Hi Adriaan


Please consider ON-ing the linter before merging. 
Also, you can push a dummy commit to trigger the linter after ON-ing it. 
Check my response in this https://github.com/adriaanbd/magicbooks/pull/1#discussion_r365832621

It's a 👍🏾 from me!

### :heavy_check_mark: Approved!


Good luck with your next milestones. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: .stickler.yml (hunk 2 above)
Comment (adriaanbd) [reply .stickler.yml:4]: Yes you are right. I thought they were ON. Thanks for pointing this out. Finally, it all makes sense.

Comment (adriaanbd) [reply .stickler.yml:4]: It works now, thanks!

**Pull request merged**